package.json

```diff
@@ -8,6 +8,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.11.2",
     "@fortawesome/react-fontawesome": "^0.1.8",
     "@reduxjs/toolkit": "^1.5.1",
+    "@tapis/tapis-typescript-streams": "^0.0.1",
```

Comment:
Is tapis-typescript-streams part of the main `@tapis/tapis-typescript` package? This should result in a version bump of tapis-typescript to 0.0.10 in https://github.com/tapis-project/tapis-typescript rather than an inclusion of the subpackage.